config/routes.rb

```diff
@@ -79,6 +79,10 @@
         memberable
         academy_urn_updateable
       ]
+    resources :projects, except: :destroy do
+      get "confirm_delete", on: :member, action: :confirm_delete, as: :confirm_delete
+      delete "/", on: :member, action: :delete, as: :delete
```

Comment:
I don't know if having this as `DELETE` is right? It doesn't actually delete anything?

Reply:
True, it neither deletes nor destroys! Should we just have a POST or PUT you reckon?

Reply:
Maybe `PATCH`? It updates the `state` value of the record with a fixed value?

Reply:
Good call! PATCH it is.